playpen/ansible/dev-playbook.yml

```diff
@@ -6,4 +6,5 @@
     - core
     - db
     - qpidd
-    - dev
\ No newline at end of file
+    - dev
+    - lazy
```

Comment:
Oops. I guess I noticed, forgot, and noticed again. Clearly I already added it to both here. :coffee: